Source/Core/VideoBackends/OGL/PostProcessing.cpp

```diff
@@ -27,7 +27,7 @@ static const char s_vertex_shader[] =
 	"void main(void) {\n"
 	"	vec2 rawpos = vec2(gl_VertexID&1, gl_VertexID&2);\n"
 	"	gl_Position = vec4(rawpos*2.0-1.0, 0.0, 1.0);\n"
-	"	uv0 = rawpos * src_rect.zw + src_rect.xy;\n"
+	"	uv0 = vec2(mix(src_rect.xy, src_rect.zw, rawpos));\n"
```

Comment:
I changed it because the old code was hardcoded except frames in opengl coordinate space (the opengl version of the virtual/real xfb code would flip it). Was impossible to set a src_rect that flipped things to the correct orientation.

I used mix because that's what the equivalent copy shader in TextureCache.cpp used. With only 4 vertices I'd really doubt you notice the overhead of mix.